src/Acuminator/Acuminator.Tests/Tests/StaticAnalysis/LongOperationStart/LongOperationInDataViewDelegateTests.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Acuminator.Analyzers.StaticAnalysis;
+using Acuminator.Analyzers.StaticAnalysis.PXGraph;
+using Acuminator.Analyzers.StaticAnalysis.LongOperationStart;
+using Acuminator.Tests.Helpers;
+using Acuminator.Tests.Verification;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Xunit;
+using System.Threading.Tasks;
+
+namespace Acuminator.Tests.Tests.StaticAnalysis.LongOperationStart
+{
+    public class LongOperationInDataViewDelegateTests : DiagnosticVerifier
```

Comment:
Please test some other cases:

1. Delegate with parameters
2. Static delegate
3. Delegate passed to `PXView`'s constructor (if this scenario is supported)

Reply:
1. Ok
2. Ok
3. No, it is not supported